lib/widgets/pages/tickets/shop_section.dart

```diff
@@ -0,0 +1,130 @@
+part of 'tickets_page.dart';
+
+class ShopSection extends StatelessWidget {
+  const ShopSection({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.all(8.0),
+      child: Column(
+        children: [
+          Align(
+            alignment: Alignment.centerLeft,
+            child: Text(Strings.shopText, style: AppTextStyle.sectionTitle),
+          ),
+          GridView.count(
+            childAspectRatio: 3 / 2,
```

Comment:
The widget tree of this widget should be changed in another branch ASAP.
Most importantly, `<something>Card`s should extend `CardBase`.
Also, we can't determine `Card`s' size (or aspect ratio) beforehand; they should be able to decide their own size. Will require us to not use `GridView`.